components/org.apache.stratos.rest.endpoint/src/main/java/org/apache/stratos/rest/endpoint/api/StratosApiV41Utils.java

```diff
@@ -2751,15 +2751,16 @@ public static void addTenant(org.apache.stratos.common.beans.TenantInfoBean tena
         UserRegistry userRegistry = (UserRegistry) PrivilegedCarbonContext.getThreadLocalCarbonContext().
                 getRegistry(RegistryType.USER_GOVERNANCE);
         if (userRegistry == null) {
-            log.error("Security alert! User registry is null. A user is trying create a tenant "
-                    + " without an authenticated session.");
-            throw new RestAPIException("Security alert! User registry is null. A user is trying create a tenant "
-                    + " without an authenticated session.");
+            String msg="Security alert! User registry is null. A user is trying create a tenant "
```

Comment:
If this is a security alert, IMO we should not raise it to the client. It can be logged in the backend and raise a different error message to the client.